launchable/commands/record/commit.py

```diff
@@ -54,14 +58,14 @@ def exec_jar(source, max_days, dry_run):
     https_proxy = os.getenv("HTTPS_PROXY")
     proxy_option = _build_proxy_option(https_proxy) if https_proxy else ""
 
-    os.system(
-        "{} {} -jar \"{}\" ingest:commit -endpoint {} -max-days {} {} {} {} {}"
-        .format(java, proxy_option, jar_file_path,
-                "{}/intake/".format(base_url), max_days,
-                "-audit" if Logger().logger.isEnabledFor(LOG_LEVEL_AUDIT) else "",
-                "-scrub-pii",
-                "-dry-run" if dry_run else "",
-                source))
+    subprocess.check_output("{} {} -jar \"{}\" ingest:commit -endpoint {} -max-days {} {} {} {} {}"
```

Comment:
os.system method doesn't throw an Exception when the command fails.
So I change to use subprocess.check_output instead of os.system